src/DssSpell.t.sol

```diff
@@ -30,6 +31,11 @@ contract DssSpellTest is DSTest, DSMath {
     // this needs to be updated
     uint256 constant SPELL_CREATED = 1606152054;
 
+    // Previous spell; supply if there is a need to test prior to its cast() function being called on mainnet.
+    address constant PREV_SPELL = address(0x8D602692eE4b5f0ec33A22fe6547822377FDCc4c);
+    // Time to warp to in order to allow the previous spell to be cast; ignored if PREV_SPELL is SpellLike(address(0)).
+    uint256 constant PREV_SPELL_EXECUTION_TIME = 1612108914;
```

Comment:
Personally, I like defining these next to their usage.

Reply:
When they need to be updated every week and by a different person it becomes a chore though, since they have to scan the whole file and make edits throughout. It kind of breaks the pattern where other modifications are made in one place.

Reply:
Arguably `MAINNET_SPELL` could be defined with its usage as well, rather than putting both values up here. Not that it matters too much, I suppose.